modules/network-observability-create-network-policy.adoc

```diff
@@ -6,7 +6,49 @@
 :_mod-docs-content-type: PROCEDURE
 [id="network-observability-network-policy_{context}"]
 = Creating a network policy for Network Observability
-You might need to create a network policy to secure ingress traffic to the `netobserv` namespace. In the web console, you can create a network policy using the form view.
+If you want to further customize the network policy for the `netobserv` namespace, you must disable the managed installation of the policy from the `FlowCollector` CR, and create your own. You can use the network policy resource that is enabled from the `FlowCollector` CR as a starting point:
```

Comment:
Actually I just remember that we create 2 network policies, one for netobserv and another for netobserv-privileged namespaces. We should call that out:

```suggestion
If you want to further customize the network policies for the `netobserv` and `netobserv-privileged` namespaces, you must disable the managed installation of the policy from the `FlowCollector` CR, and create your own. You can use the network policy resources that are enabled from the `FlowCollector` CR as a starting point:
```

Reply:
here's the one created in netobserv-privileged:

```yaml
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: netobserv
  namespace: netobserv-privileged
spec:
  ingress:
  - from:
    - namespaceSelector:
        matchLabels:
          kubernetes.io/metadata.name: openshift-monitoring
  podSelector: {}
  policyTypes:
  - Ingress

```